website_sale_product_subscribe_notify/models/product_template.py

```diff
@@ -24,12 +24,15 @@ def write(self, vals):
                     partner_ids = rec.get_product_category_followers_ids()
                     partners = rec.env['res.partner'].browse(partner_ids)
                     list_price = "%d" % int(rec.list_price)
+                    limit_recipient = rec.env['ir.default'].get('res.config.settings', 'email_recipient_limit')
+                    number_of_loop = round(len(partners)/limit_recipient)
+
                     ctx.update({
                             'website_published_update': vals.get(
                                 "website_published"),
                             'list_price_update': vals.get("list_price"),
                             'description_sale_update': vals.get("description_sale"),
-                            'partner_ids': ','.join([str(partner.id) for partner in partners]),
+                            'partner_ids': ','.join([str(partner.id) for partner in partners[:limit_recipient]]),
```

Comment:
@TimLai125  For looping and cut the list, I might not exactly know on how to use that. Do you have any more suggestion?

Reply:
The structure of this part should be:
```
for n in range(1, number_of_loop):
    ctx.update({...})
    template.with_context(ctx).send_mail(rec.id)
```
In each loop, you need to figure out which part of the `partners` you need and express it in terms of `n` and `limit_recipient`.